mentat/vision/vision_manager.py

```diff
@@ -0,0 +1,49 @@
+import datetime
+import os
+from pathlib import Path
+
+import attr
+from selenium import webdriver
+from selenium.webdriver.chrome.service import Service
+from webdriver_manager.chrome import ChromeDriverManager
+
+from mentat.utils import mentat_dir_path
+
+
+@attr.define
+class VisionManager:
+    driver: webdriver.Chrome | None = attr.field(default=None)
+
+    def init(self) -> None:
+        "This opens a browser with no page open so we don't want to call it until the user actually wants it."
+        if self.driver is None:
+            service = Service(ChromeDriverManager().install())
+            self.driver = webdriver.Chrome(service=service)
+
+    def open(self, path: str) -> None:
+        self.init()
```

Comment:
Does this open a new browser every time and does it close the browser afterwards or not?

Reply:
It opens in the same browser. The browser is automatically closed when mentat quits (or the user can close it themselves).